### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Nataliia-Popkova**
Add new workflow for suite with dms in ON mode. Create new folder with tests for suite-dms.

## Checklist
- [ x ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: .github/workflows/suite-regression-api-dms-on.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+                  python3 -m pip install -U robotframework-jsonlibrary
+                  python3 -m pip install -U robotframework-databaselibrary
+                  python3 -m pip install PyMySQL
+            - name: Install docker-compose
+              run: |
+                  sudo curl -L "https://github.com/docker/compose/releases/download/2.12.0/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+                  sudo chmod +x /usr/local/bin/docker-compose
+            - name: Install Robot tests
+              run: |
+                  git clone https://github.com/spryker/robotframework-suite-tests.git --single-branch --branch dev robotframework-tests
```
Comment (dmiseev) [inline .github/workflows/suite-regression-api-dms-on.yml:46]: master or dev branch?

File: .github/workflows/suite-regression-api-dms-on.yml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+            - name: Install Project
+              continue-on-error: false
+              run: |
+                  git clone https://github.com/spryker/docker-sdk.git ./docker
+                  docker/sdk boot -v deploy.ci.api.mariadb.dynamic-store.yml
```
Comment (dmiseev) [inline .github/workflows/suite-regression-api-dms-on.yml:51]: In the original `suite-regression-api.yml` we are using `deploy.ci.acceptance.mariadb.robot.yml`.
So are you sure that you picked the correct deploy file?
`deploy.ci.api.mariadb.dynamic-store.yml` -> `deploy.ci.acceptance.mariadb.dynamic-store.yml`

File: .github/workflows/suite-regression-api-dms-on.yml (hunk 1 above)
Comment (Nataliia-Popkova) [reply .github/workflows/suite-regression-api-dms-on.yml:46]: Dev branch - i need them firs in test mode (will merge new changes to dev)

File: .github/workflows/suite-regression-api-dms-on.yml (hunk 2 above)
Comment (Nataliia-Popkova) [reply .github/workflows/suite-regression-api-dms-on.yml]: fixed

**Review by dmiseev — APPROVED** [review]

**Pull request merged**